docker/compose/indexer/docker-compose.hasura.yml

```diff
@@ -0,0 +1,47 @@
+services:
+  graphql-engine:
+    image: hasura/graphql-engine:v2.40.0
+    ports:
+      - "8085:8085"
+    restart: always
+    environment:
```

Comment:
So, does the existing Docker compose work for the rest of the indexer. That is, as long as we provide the same postgres connection here and there, are we good? Or, are we still missing the `indexer-processor`?

Reply:
I think it's better to provide 2 docker scripts, one for Hasura and one for the indexer. I can use the same Ip env variable for the indexer docker script.